doc/manuals/user/v1_v2_coexistence.md

```diff
@@ -51,3 +51,11 @@ will automatically replace these cases with the string value `none` (which is th
 The `orderBy` parameter defined for NGSIv2 can be used also in NGSIv1 queryContext operation (see
 details in the [pagination documentation](pagination.md). However, note that the "geo:distance"
 order can be used only in NGSIv2.
+
+## NGSIv1 notification with NGSIv2 subscriptions
+
+NGSIv2 allows several notification modes depending on the `attrsFormat` field associated to the
+subscription. Appart of the values described in the NGSIv2 specification, Orion also support
+`legacy` value in order to send notifications in NGSIv1 format. This way, users can have the
```

Comment:
Indeed, this `legacy` mode is the current way Orion works (as in Orion 0.27.0) for NGSIv2 subscriptions.

NTC (just a remark)